### PR TITLE
Count

### PR DESCRIPTION
**Pull request by nate-e-97**
Count command works

Also removed db/pool because it has become deprecated

### CODE REVIEW

File: discord/commands/count.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      return;
+  }
+
+  switch (CommandStruct.args[0]) {
+        case 'red':
+            ComposeCount('dsrred', CommandStruct, 5)
+            break;
+
+        case 'orange':
+        case 'pink':
```
Comment (ghost) [inline discord/commands/count.js:23]: why are we here? just to suffer?

File: discord/commands/count.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
+/** This is a test command to show what command files should look like */
```
Comment (ghost) [inline discord/commands/count.js:1]: we should probably clean up these comments, but it doesn't matter much either way

File: reddit/init.js [hunk 3]
```diff
@@ -9,4 +9,6 @@ const redditClient = new snoowrap({
     password: process.env.REDDIT_PASSWORD
 })
 
+redditClient.config({ continueAfterRatelimitError: true })
```
Comment (ghost) [inline reddit/init.js:12]: ```Determines whether snoowrap should queue API calls if reddit's ratelimit is exceeded. If set to true when the ratelimit is exceeded, snoowrap will queue all further requests, and will attempt to send them again after the current ratelimit period expires (which happens every 10 minutes). If set to false, snoowrap will simply throw an error when reddit's ratelimit is exceeded.```

that's it, I'm moving from praw to snoowrap

**Review by ghost — APPROVED** [review]
Some comments I'm not sure if are necessary, but the command is amazing and works

one thing: do you know what will happen if multiple instances are running on separate machines? in case several people are using the same testing credentials

File: discord/commands/count.js (hunk 2 above)
Comment (nate-e-97) [reply discord/commands/count.js:1]: how obvious is it that i'm just copying and pasting ping whenever i make a new command

**Pull request merged**